src/sentry/receivers/outbox/control.py

```diff
@@ -25,6 +27,7 @@
 from sentry.organizations.services.organization import RpcOrganizationSignal, organization_service
 from sentry.receivers.outbox import maybe_process_tombstone
 from sentry.sentry_apps.services.app.service import get_by_application_id, get_installation
+from sentry.services.hybrid_cloud.relocation_export import region_relocation_export_service
```

Comment:
The `sentry/services/hybrid_cloud` module was removed in master. Could this be moved to `sentry.backup.services` with the existing import_export service?

Reply:
See below, resolved.